server/app.js

```diff
@@ -17,8 +19,21 @@ app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(join(__dirname, "public")));
 
+db.mongoose.connect("mongodb://localhost:27017/userDB", {useNewUrlParser: true});
```

Comment:
All of these DB things should be changed to use @johnsciutto's code.  We can merge that in now.

Reply:
This line of code for my testing purpose. Anyway, I am going to delete that one. Also, userModel folder is unnecessary. It is only for my testing purpose. I think we can remove that one after @johnsciutto completes his curd operation. I am going to use his methods in my code.

Reply:
Hi @shahgrishma123. I'm done with all the CRUD methods, I'm going to merge into this branch now. I added detailed instructions in the README for all the database-related methods. 
Also, because you're going to delete the "userModel" folder, I moved all the functions that are still needed to the "models" folder and to the "utilities" folder.